docs/modules/secret-operator/pages/secretclass.adoc

```diff
@@ -90,7 +90,7 @@ Native support for customizing certificate lifetimes in Stacklet CRDs might be a
 
 Certificate authorities also have a limited lifetime, and need to be rotated before they expire to avoid cluster disruption.
 
-If configured to provision its own CA (`autoTls.ca.autoGenerate`), the Secret Operator will create CA certificates that are valid for 2 years (`autoTls.ca.caCertificateLifetime`), and initiate rotation once less than half of that time remains.
+If configured to provision its own CA (`autoTls.ca.autoGenerate`), the Secret Operator will create CA certificates that are valid for 1 year (`autoTls.ca.caCertificateLifetime`), and initiate rotation once less than half of that time remains.
```

Comment:
Does it make sense to be precise here and say 365 days instead?

Reply:
I'd argue that if the distinction matters then we've already failed.. but sure.

Reply:
I approved the PR. I don't have strong feelings, it was an honest question. I'll leave the decision up to you.

Reply:
f0642f51440b4a351c5142e58500f234509a4efd